app/services/asset_manager/attachment_updater/replacement_id_updates.rb

```diff
@@ -6,19 +6,24 @@ def self.call(attachment_data)
 
     Enumerator.new do |enum|
       enum.yield AssetManager::AttachmentUpdater::Update.new(
-        attachment_data, attachment_data.file, replacement_legacy_url_path: replacement.file.asset_manager_path
-      )
+                   attachment_data,
+                   attachment_data.file,
+                   replacement_legacy_url_path: replacement.file.asset_manager_path
+                 )
 
       if attachment_data.pdf?
-        if replacement.pdf?
-          enum.yield AssetManager::AttachmentUpdater::Update.new(
-            attachment_data, attachment_data.file.thumbnail, replacement_legacy_url_path: replacement.file.thumbnail.asset_manager_path
+        replacement_legacy_url_path = if replacement.pdf?
+                                        replacement.file.thumbnail.asset_manager_path
+                                      else
```

Comment:
You too are a fan of hanging indents, I see. 😇

Reply:
No really... it's just that the linter won't accept anything else!